manifests/install/rabbitmqadmin.pp

```diff
@@ -25,13 +27,14 @@
     $sanitized_ip = $management_ip_address
   }
 
-  staging::file { 'rabbitmqadmin':
-    target      => "${rabbitmq::rabbitmq_home}/rabbitmqadmin",
-    source      => "${protocol}://${sanitized_ip}:${management_port}/cli/rabbitmqadmin",
-    curl_option => "-u \"${default_user}:${default_pass}\" -k ${curl_prefix} --retry 30 --retry-delay 6",
-    timeout     => '180',
-    wget_option => '--no-proxy --no-check-certificate',
-    require     => [
+  archive { 'rabbitmqadmin':
+    path           => "${rabbitmq::rabbitmq_home}/rabbitmqadmin",
```

Comment:
Not sure if it's better to update this to just stage it, drop it in its final destination upon successful retrieval, and then clean up the tmp file vs. having the file resource?

Reply:
I think the current behavior is totally valid